### PR TITLE
[hack] Add script to collect Windows network trace

### PR DESCRIPTION
**Pull request by sebsoto**
Captures .etl traces which can be converted into a .pcapng file, to be
consumed by either Wireshark or other programs.

### CODE REVIEW

**Review by aravindhp — APPROVED** [review]
Thanks for doing this, @sebsoto. 

/approve

File: docs/TROUBLESHOOTING.md [hunk 1]
```diff
@@ -85,6 +85,17 @@ Alternatively, the Docker event logs can be viewed using SSH with the following
   PS C:\Users\username> Get-EventLog -LogName Application -Source Docker
   ```
```
Comment (VaishnaviHire) [inline docs/TROUBLESHOOTING.md:87]: Can you add the following here -
`Requires SSH [bastion](https://github.com/eparis/ssh-bastion) deployed on cluster`

**Review by VaishnaviHire — COMMENTED** [review]
Thanks @sebsoto . 
Mostly LGTM.  PTAL at my suggestion

**Pull request merged**